x-pack/plugins/apm/server/routes/services/get_services_detailed_statistics/get_service_transaction_detailed_statistics.ts

```diff
@@ -91,33 +93,42 @@ export async function getServiceTransactionDetailedStatistics({
           },
         },
         aggs: {
-          services: {
-            terms: {
-              field: SERVICE_NAME,
+          sampled: {
```

Comment:
for the sake of consistency can we use either `sample` or `sampled`? 🙏

Reply:
done! changed `sampled` to `sample`.